common/predictive-text/worker/worker-interfaces.ts

```diff
@@ -135,8 +135,54 @@ interface LMLayerWorkerState {
  * The model implementation, within the Worker.
  */
 interface WorkerInternalModel {
+  /**
+   * Processes `config` messages, configuring the newly-loaded model based on the host
+   * platform's capability restrictions.
+   * 
+   * This allows the model to configure its suggestions according to what the platform
+   * allows the host to actually perform - for example, if post-caret deletions are not
+   * supported, no suggestions requiring this feature should be produced by the model.
+   * 
+   * Returns a `Configuration` object detailing the capabilities the model plans to
+   * actually utilize, which must be more restrictive than those indicated within
```

Comment:
Small nitpick: "must be **more** restrictive" than the capabilities? Isn't this more like "can match the capabilities OR be more restrictive"?